config.js

```diff
@@ -54,7 +54,7 @@ Config.prototype.getDir = function (name, opts, done) {
 };
 
 function merge_config (defaults, overrides, type) {
-    if (type === 'ini' || type === 'json' || type === 'yaml') {
+    if (type === 'ini' || type === 'hjson' || type === 'json' || type === 'yaml') {
```

Comment:
after a few conditionals, it gets cleaner to convert this to a switch/case. Agree?

Reply:
Yes, will do